### PR TITLE
support: Hide content of page list timeline

### PR DESCRIPTION
**Pull request by mudana-grune**
https://youtrack.weseek.co.jp/issue/GW-7872
- Hide content of page list timeline
- Hide content of page list timeline on full screen mode

### CODE REVIEW

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 1]
```diff
@@ -30,33 +30,24 @@ context('Access to pagelist', () => {
...
-    cy.getByTestid('page-duplicate-modal').should('be.visible').within(() => {
-      cy.get('.modal-footer > button').click();
-    });
     cy.get('body').type('{esc}');
-    cy.getByTestid('pageListButton').click({force: true});
-    cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
-      cy.get('.list-group-item').eq(0).within(() => {
-        cy.screenshot(`${ssPrefix}5-duplicated-page`);
-      });
-    });
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:42]: what is the reason for removing these?

Comment (mudana-grune) [reply packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:42]: Because previously duplicate pages made the order and contents of the page list/timeline change, so I think skipping it will reduce page list changes. I'll revert it while minimizing changes to the page list/timeline

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 2]
```diff
@@ -56,7 +56,11 @@ context('Access to pagelist', () => {
       cy.get('button.close').eq(1).click();
     });
 
-    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {capture: 'viewport'});
+    // Hide release & license on home page , page-list count and page creation & date
+    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {
+      capture: 'viewport',
+      blackout: ['[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt=true]', '[data-testid="pageListButton"] > span']
+    });
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:63]: Why is this diff here?
It looks like this is not related to `page list timeline`

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 3]
```diff
@@ -86,10 +90,18 @@ context('Access to timeline', () => {
       cy.get('.nav-title > li').eq(1).find('a').click();
       cy.get('button.close').eq(0).click();
     });
-    cy.screenshot(`${ssPrefix}2-timeline-list-fullscreen`, {capture: 'viewport'});
+
+    // Hide release & license on home page , page-list count and page creation & date
+    cy.screenshot(`${ssPrefix}2-timeline-list-fullscreen`, {
+      capture: 'viewport',
+      blackout: ['.card-timeline:eq(0) > .card-body > #wiki > p > a > img']
+    });
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:98]: This one looks like the whole content is changed and I don't know what are being blackouted here.

Before
image

After
image

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 4]
```diff
@@ -55,8 +55,6 @@ context('Access to pagelist', () => {
     cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
       cy.get('button.close').eq(1).click();
     });
-
-    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {capture: 'viewport'});
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:59]: what is the reason you deleted this?
it seems tthat the goal for this test is: Successfully expand and `close modal`

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 5]
```diff
@@ -87,9 +85,5 @@ context('Access to timeline', () => {
       cy.get('button.close').eq(0).click();
     });
     cy.screenshot(`${ssPrefix}2-timeline-list-fullscreen`, {capture: 'viewport'});
-    cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
-      cy.get('button.close').eq(1).click();
-    });
-    cy.screenshot(`${ssPrefix}3-close-modal`, {capture: 'viewport'});
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:93]: this one too

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts (hunk 4 above)
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:59]: Is this not needed to assure that the modal is closed?

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts (hunk 5 above)
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:93]: same question as above

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 6]
```diff
@@ -55,8 +55,10 @@ context('Access to pagelist', () => {
     cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
       cy.get('button.close').eq(1).click();
     });
-
-    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {capture: 'viewport'});
+    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {
+      capture: 'viewport',
+      blackout: ['[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt=true]']
+    });
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:61]: Does this blackout only works when taking screenshot if you nest the blackout option inside `cy.screenshot` ?

File: packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts [hunk 7]
```diff
@@ -90,6 +92,9 @@ context('Access to timeline', () => {
     cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
       cy.get('button.close').eq(1).click();
     });
-    cy.screenshot(`${ssPrefix}3-close-modal`, {capture: 'viewport'});
+    cy.screenshot(`${ssPrefix}3-close-modal`, {
+      capture: 'viewport',
+      blackout: ['[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt=true]']
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts:97]: Did you need blackout here?
Do the `img` change before you open and after you close modal?

**Comment by Yohei-Shiina** [comment]
I think you are forgetting codes to fix this one
https://growi-vrt-snapshots.s3.amazonaws.com/5a9da36ab9de39921305bbf21b4471ab75d8d41c/index.html?id=changed-20-basic-features/access-to-pagelist.spec.ts/access-to-timeline-1-timeline-list.png
